src/imp/libc/process/types.rs

```diff
@@ -135,6 +135,121 @@ impl Resource {
     pub const Rss: Self = Self::As;
 }
 
+/// A signal number for use with [`kill`].
+///
+/// [`kill`]: crate::process::kill
+#[cfg(not(target_os = "wasi"))]
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+#[repr(i32)]
+pub enum Signal {
+    /// `SIGHUP`
+    Hup = c::SIGHUP,
```

Comment:
This is a taste thing but I think this would be better if the elements stayed uppercase.  The whole unix "let's drop the vowels to save space" makes having these rendered as capitalized words read strangely to me.  `Alrm` looks particularly odd for example.  Even if it's `Signal::Alrm`.  I think it's more obviously "Unix" if it's `Signal::ALRM`.  This is how the nix crate has it at least.

Reply:
Good idea! I've now opted to go for `Signal::Alarm`, and similar for `Abrt`, `Chld`, and a few similar ones. This is consistent with rustix things like `CREATE` for `O_CREAT` and others.

Rustix overall aims to balance human-friendliness with C familiarity, and these "missing vowels" cases are easy places where we can add friendliness without losing much familiarity.

Reply:
Ah, I should also mention, the reason for avoiding all-caps is to be consistent with the [Rust convention](https://rust-lang.github.io/api-guidelines/naming.html) for enum variant names.

Overall, rustix is aiming for a balance between Rust-friendliness, human-friendliness, and C familiarity.

Reply:
Ah, OK.  That makes sense.